webpack.common.js

```diff
@@ -1,36 +1,27 @@
-const path = require('path');
-
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin');
 const PreloadWebpackPlugin = require('preload-webpack-plugin');
 
 module.exports = {
   mode: 'development',
-  entry: './src/index.js',
+  entry: {
+    main: './src/index.js',
+    vendor: './src/vendor.js'
+  },
   module: {
     rules: [{
         test: /\.txt$/,
         use: 'raw-loader'
       },
-      {
-        test: /\.html$/,
-        use: [{
-          loader: 'html-loader',
-          options: {
-            minimize: true
-          }
-        }]
-      },
       {
         test: /\.(jpe?g|png|gif|svg)$/,
         use: [{
           loader: 'file-loader',
           options: {
             name: '[name].[ext]',
             outputPath: 'images/',
-            publicPath: 'images/'
+            publicPath: '/images/'
```

Comment:
How come this makes a difference? I thought slash and no slash achieve the same thing. Have you tested the use case?

Cheers

Reply:
index.html file gets generated in dist directory and images are kept in the same directory.

e.g 
dist/
    index.html
    images/

in this scenario no slash is fine as the path for image files will be like src="images/123.png" as file and images are in same diectory.

now consider this case. when user has multiple html files in different folder like

dist/
    index.html
    images/
    privacy-policy/
        index.html 

now if index.html inside privacy-policy folder has image path like src="images/123.png" it wont be able to load image as there is no image folder in privacy policy directory. 

so if you add a slash path for images will be resolved like src="https://yourwebsite.com/images/123.com" which will work for any html files in any directory.

Reply:
Makes sense, thanks.